src/arbitrage.py

```diff
@@ -80,6 +83,48 @@ class API(object):
     def __init__(self, args=None):
         self.name = args
 
+    def get(self, stock):
+        exps = self.get_expiries(stock)
+        spot = self.get_spot_price(stock)
+        premiums = {}
+        for exp in exps:
+            premiums[exp] = self.get_option_premiums(stock, exp, spot)
+        return StockData(stock, exps, spot, premiums)
+
+    def get_expiries(self, stock):
+        if self.name == "tradier":
+            option_expiries = requests.get('https://sandbox.tradier.com/v1/markets/options/expirations',
+                                           params={'symbol': stock, 'includeAllRoots': 'true', 'strikes': 'false'},
+                                           headers={'Authorization': 'Bearer Bfo8MwBCA6lFOqWSdWIe1Ke7IigA',
```

Comment:
params={'symbol': stock, 'includeAllRoots': 'true', 'strikes': 'false'},
headers={'Authorization': 'Bearer Bfo8MwBCA6lFOqWSdWIe1Ke7IigA',

This is used a lot here.

Can you make this a constant instead of repeating so much?

Reply:
fixed